content/spin/v2/variables.md

```diff
@@ -11,60 +11,72 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/variables.
+Variables are surfaced to a specific component by adding a `[component.(name).variables]` section to the component and referencing them within it. The `[component.(name).variables]` section contains a mapping of component variables and values. Entries can be static (like `api_version` below) or reference an updatable application variable (like `token` below) using [mustache](https://mustache.github.io/)-inspired string templates. Only components that explicitly use variables in their configuration section will get access to them. This enables only exposing variables (such as secrets) to the desired components of an application.
 
 <!-- @nocpy -->
 
 ```toml
-[component.cart.variables]
-password = "\{{ secret }}"
-api_host = "https://my-api.com"
+[component.(name).variables]
+token = "\{{ api_token }}"
+api_uri = "\{{ api_uri }}"
+api_version = "v1"
 ```
 
-When a component variable references an application variable, its value will dynamically update as the application variable changes. For example, if the `secret` variable is provided using the [Spin Vault provider](./dynamic-configuration.md#vault-application-variable-provider), it can be updated by changing the value in HashiCorp Vault. The next time the component gets the value of `password`, the latest value of `secret` will be returned by the provider. See the [next section](#using-variables-from-applications) to learn how to use Spin's configuration SDKs to get configuration variables within applications.
+Variables can also be surfaced to other sections of the application manifest that benefit from dynamic configuration. For example, the `allowed_outbound_hosts` can be dynamically configured using variables as follows:
 
-An application manifest with a `secret` variable and a component that uses it would look similar to the following:
+<!-- @nocpy -->
+
+```toml
+[component.(name)]
+allowed_outbound_hosts = [ "\{{ api_uri }}" ]
+```
+
+When a component variable references an application variable, its value will dynamically update as the application variable changes. For example, if the `api_token` variable is provided using the [Spin Vault provider](./dynamic-configuration.md#vault-application-variable-provider), it can be updated by changing the value in HashiCorp Vault. The next time the component gets the value of `token`, the latest value of `api_token` will be returned by the provider. See the [next section](#using-variables-from-applications) to learn how to use Spin's configuration SDKs to get configuration variables within applications.
```

Comment:
This refers specifically to component variables referencing application variables, so goes best either with the section on component variables (where it was before), or in a new section separated from the "variables in other parts of the manifest" stuff.

I think I'd prefer to move it back, and then we can add something here that variables in other parts of the manifest are substituted at application load time and not dynamically updated while the application is running.  Open to thoughts though!

Reply:
Good call. I agree that it fits better right after the component section.